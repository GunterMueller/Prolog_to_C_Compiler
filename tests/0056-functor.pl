main :-
	functor(123, 123, 0),
	functor(abc, abc, 0),
	functor([1,2], '.', 2),
	functor(foo(bar, 1), foo, 2),
	functor([], [], 0),
	functor(X, foo, 8),
	functor(X, foo, 8),
	display(X), nl.
