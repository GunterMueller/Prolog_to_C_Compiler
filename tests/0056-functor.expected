foo(_1, _2, _3, _4, _5, _6, _7, _8)
